drivers/flash/Kconfig

```diff
@@ -84,6 +84,12 @@ config FLASH_PAGE_LAYOUT
 	help
 	  Enables API for retrieving the layout of flash memory pages.
 
+config FLASH_RESET_REGISTERS
```

Comment:
how about using name as FLASH_RESET? and rest of the function names also similar name

Reply:
Done, Thanks